ROADMAP.md

```diff
@@ -1,102 +1,126 @@
+* [Blogpost] <a href="https://aws.amazon.com/blogs/compute/better-together-aws-sam-and-aws-cdk/">Better together: AWS SAM and AWS CDK</a>
+* [Videos] <a href="https://www.youtube.com/@CDK-Live">CDK Live!</a>
+* [CDK Day] <a href="https://www.youtube.com/watch?v=qlUR5jVBC6c">Track 1 (English) for CDK Day 2023</a>
+* [CDK Day] <a href="https://www.youtube.com/watch?v=b-nSH18gFQk">Track 2 (English) for CDK Day 2023</a>
+* [CDK Day] <a href="https://www.youtube.com/watch?v=ZAQC-cOXL4M">Track 3 (Spanish) for CDK Day 2023</a>
+* [re:Invent Content] Search through <a href="https://www.youtube.com/@amazonwebservices">all content here</a>
 
-**Q: Is every feature on the roadmap?**
+## 2024 Updates
 
-A: The AWS Cloud Development Kit roadmap provides transparency on our priority for adding new programming languages,
-developer experience improvements, and service coverage in the AWS Construct Library. The AWS CDK toolkit and AWS
-Construct Library are such a large surface areas we are intentionally keeping the roadmap at a high-level, so not every
-CDK feature request will appear on the roadmap. Instead, the roadmap will include a tracking issue
-for each deliverable that provides a feature overview and contains links to relevant, more granular issues and pull
-requests. If you want to track the status of a specific issue or pull request, you can do so by monitoring that work
-item in the [aws-cdk] GitHub repository.
+Stay tuned for more updates for 2024!
 
-**Q: What is a tracking issue?**
+## Disclaimer
 
-A: We create a tracking issue for each CDK feature, AWS Construct Library module, and jsii-supported programming language. Tracking issues provide a brief summary of the feature and a consolidated view of the work scoped for the release. They include links to design documentation, implementation details, and relevant issues. Tracking issues are living documents that start from a basic template and grow more robust over time as we experiment and learn. You can easily find tracking issues by filtering on the [management/tracking label](https://github.com/aws/aws-cdk/labels/management%2Ftracking).
+The AWS CDK team values feedback and guidance from its community of users, although final decisions on inclusion into the project will be made by AWS. We determine the high-level direction for our open roadmap based on customer feedback and popularity (👍🏽 and comments), security and operational impacts, and business value. Where features don’t meet our goals and longer-term strategy, we will communicate that clearly and openly as quickly as possible with an explanation of why the decision was made.
 
-**Q: How can I provide feedback on the roadmap or ask for more information about a feature?**
+## FAQs
 
-A: Please open an issue! Or engage by 👍 existing ones.
+Q: Why did you build an open roadmap?
```

Comment:
The QandA section doesn't render the way you're expecting, I think. You need a space between the lines for a space to show up in the markdown render.